api/src/test/java/io/grpc/GlobalInterceptorsTest.java

```diff
@@ -0,0 +1,163 @@
+/*
+ * Copyright 2022 The gRPC Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.grpc;
+
+import static com.google.common.truth.Truth.assertThat;
+import static org.junit.Assert.fail;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import org.junit.Test;
+
+public class GlobalInterceptorsTest {
+
+  private static final ServerStreamTracer.Factory DUMMY_SERVER_TRACER =
```

Comment:
Can we keep `DUMMY_SERVER_TRACER` and the 2 Noop test classes together? Either both at top or at bottom? Also consistent use. Here you are using a static instance defined using anonymous inline class but in the other case defining test classes